### ISSUE TITLE
Indexing not cached ?

### ISSUE THREAD

**Issue by Erwyn** [zobo/php-language-server#14]
Maybe it's on my side. I simply installed the VSCode extension, it indexes and works well. But when I close and relaunch VSCode, my project is entirely indexed again. Is it normal ? Did I miss a configuration option ?

**Comment by zobo** [comment]
No, you are not doing anything wrong. It's simply implemented like this for now. Versioned vendor packages are cached, your project gets reindexed every time.
I already noted this here for myself https://github.com/zobo/vscode-php-intellisense/issues/9
Will get to that hopefully soon.

**Comment by Erwyn** [comment]
Thanks! duly noted.

I'm sorry I thought #9 was about something else entirely hence the duplicate. Do you want me to close this one or  do you want to keep it open and close it once / if it's done ?

**Comment by zobo** [comment]
It's ok, leave it for others to see. I'll close it when I'm able to do it. Thanks!
